corehq/apps/cloudcare/static/cloudcare/js/form_entry/web_form_session.js

```diff
@@ -331,7 +331,7 @@ hqDefine("cloudcare/js/form_entry/web_form_session", function () {
 
             // We revalidate any errored labels while answering any of the questions
             var erroredLabels = form.erroredLabels();
-
+            sessionStorage.answerQuestionInProgress = true;
```

Comment:
could this be on `self` instead of in `sessionStorage`?

Reply:
for where this is being checked I think it makes sense to follow the pattern of `sessionStorage.validationInProgress` . Both are checked in [shouldShowLoading](https://github.com/dimagi/commcare-hq/blob/0c8c94fce0f7eb0f759ebc23ea5cbb694e6092e2/corehq/apps/cloudcare/static/cloudcare/js/utils.js#L128) to decide if the loading indicator should be shown

Reply:
>session storage is scoped to a tab, correct?

yes, that's right